osu.Game/Overlays/Changelog/ChangelogHeader.cs

```diff
@@ -72,7 +82,21 @@ private void showBuild(ValueChangedEvent<APIChangelogBuild> e)
             AutoSizeAxes = Axes.Y,
             Children = new Drawable[]
             {
-                Streams = new UpdateStreamBadgeArea(),
+                streamsBackground = new Box
+                {
+                    RelativeSizeAxes = Axes.Both
+                },
+                new Container
+                {
+                    RelativeSizeAxes = Axes.X,
+                    AutoSizeAxes = Axes.Y,
+                    Padding = new MarginPadding
+                    {
+                        Horizontal = 85,
```

Comment:
Pretty sure this should be
```suggestion
                        Horizontal = 65,
```
<details>
<summary>Devtools screenshots</summary>

![image](https://user-images.githubusercontent.com/20418176/75592730-5a466580-5a83-11ea-8091-7cb6aefce746.png)

![image](https://user-images.githubusercontent.com/20418176/75592784-88c44080-5a83-11ea-8853-03b9e35348dc.png)
</details>

The paddings don't stack.